src/modules/users/users.controller.ts

```diff
@@ -85,4 +89,19 @@ export class UsersController {
       data: updatedUser,
     };
   }
+
+  @Put('/wishlist/:accommodationId')
+  @UseGuards(JwtAuthGuard)
+  @UpdateUserSwaggerDecorator()
+  async wishlistAccommodation(
+    @Param('accommodationId', new ParseUUIDV4Pipe()) accommodationId: string,
+    @Request() req: any,
+  ) {
+    try {
+      const userId = req.user.sub;
+      await this.usersService.wishlistAccommodation(userId, accommodationId);
+    } catch (error) {
```

Comment:
you don't need try catch. your service already throws exceptions . right now you are just catching your own exceptions.